Makefile

```diff
@@ -36,47 +37,51 @@ default: build
 # and will only work - when this tree is found on the GOPATH.
 ifeq ($(CURDIR),$(PROJECT_DIR))
 
-ifeq ($(JUJU_MAKE_GODEPS),true)
-$(GOPATH)/bin/godeps:
-	go get github.com/rogpeppe/godeps
+ifeq ($(JUJU_MAKE_GODEPS),true)  # ?? change this to JUJU_MAKE_DEP
```

Comment:
Can we change it here to JUJU_MAKE_DEP and add a snippet to assign from JUJU_MAKE_GODEPS if defined (and JUJU_MAKE_DEP is not assigned)

Reply:
yes, can do